src/helix-ui/styles/components/layouts.less

```diff
@@ -39,6 +39,7 @@ body {
 
     > #content {
       flex-grow: 1;
+      min-width: 0px;
```

Comment:
Fixes a layout issue with long, unbreakable text forcing the stage to be wider than the page.